src/api/image/bin/unsplash-download.js

```diff
@@ -18,7 +17,7 @@ const pipeline = promisify(stream.pipeline);
  * @param {string} filename - the local filename to use when writing
  */
 function downloadPhoto(url, filename) {
-  return pipeline(got.stream(url), fs.createWriteStream(filename)).then(() =>
+  return pipeline(fetch(url), fs.createWriteStream(filename)).then(() =>
```

Comment:
I think this has to be the `body` of the response to get a stream, see https://github.com/node-fetch/node-fetch#streams

Reply:
Thanks! Will do